### PR TITLE
Bitbucket Server Source: Part 3

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

Add pull_request_diffs stream

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: sources/bitbucket-server-source/src/bitbucket-server/index.ts [hunk 1]
```diff
@@ -232,6 +240,58 @@ export class BitbucketServer {
...
+      );
+      for (const pr of await prs) {
+        const {data} = await this.client[MEP].pullRequests.getDiff({
+          projectKey,
+          repositorySlug,
+          pullRequestId: pr.id,
+        });
+        try {
+          yield {
+            files: parseDiff(data).map((f) =>
```
Comment (tovbinm) [inline sources/bitbucket-server-source/src/bitbucket-server/index.ts:266]: very cool!

Comment (tovbinm) [reply sources/bitbucket-server-source/src/bitbucket-server/index.ts:266]: wanna try using `Diff.parsePatch(diffStr)` from https://www.npmjs.com/package/diff ? it seems like a more established library

Comment (cjwooo) [reply sources/bitbucket-server-source/src/bitbucket-server/index.ts:266]: We're currently using the same library that we use for Phabricator, which we know works. Let's get this merged and ready for use by our stakeholders. We can try alternative libraries afterwards.

Comment (tovbinm) [reply sources/bitbucket-server-source/src/bitbucket-server/index.ts:266]: ok, thanks. please do followup

**Review by tovbinm — APPROVED** [review]

**Pull request merged**